packages/libs/react-ui/src/components/Card/Card.css.ts

```diff
@@ -19,7 +19,6 @@ export const container = style([
     borderRadius: '$sm',
     marginY: '$6',
     border: 'none',
-    width: 'max-content',
```

Comment:
there is an expanded prop that you can use to make the notification component full width

Reply:
fixed